README.md

```diff
@@ -482,6 +482,7 @@ meant to be able to change with it.
   * <a name="ambiguous-hook-scope"></a>
     Use `:context` instead of the ambiguous `:all` scope in `before`/`after`
     hooks.
+    <a href="#etwr3">[etwr3, Ch 7]</a>
```

Comment:
How do you feel about using the same style as `link` below? E.g.:
```
<sup>[[etwr3, Chapter 7](#etwr3)]</sup>
```

Reply:
Yes, I considered that, but since they serve quite distinct purpose, I think it would be good to differentiate them:

* The `[link]` links are 'utilitarian' – they provide a way to direct people to particular sections of the document, so can be less prominent.
* The reference links are part of the narrative – they credit the origin of the guidelines, so should be more prominent.

What do you think?

Reply:
I concur.

Reply:
My concern is that it doesn't look too attractive 
![image](https://user-images.githubusercontent.com/6916/56974407-79c8e500-6b77-11e9-82ae-ddf28f5eff27.png)

How do you feel about using a blockquote instead?

![image](https://user-images.githubusercontent.com/6916/56975134-d4167580-6b78-11e9-95cc-ba62a058e319.png)

Reply:
Hmm, that seems semantically wrong, since it's not actually a quote.

Perhaps we can revisit this after #95?

Reply:
WDYT of keeping this open to remind us to add proper references, and meanwhile play with AsciiDoc?

Reply:
c

Reply:
Sure, it's not urgent.